cmd/new.go

```diff
@@ -88,14 +89,25 @@ func NewRecipeCmd() *cobra.Command {
 				}
 			}
 
-			return generator.Recipe(args[0], extractor, sinkList, procList)
+			return generator.Recipe(generator.RecipeParams{
+				Name:       args[0],
+				Source:     extractor,
+				Scope:      scope,
+				Sinks:      sinkList,
+				Processors: procList,
+			})
 		},
 	}
 
 	cmd.Flags().StringVarP(&extractor, "extractor", "e", "", "Type of extractor")
+	cmd.Flags().StringVarP(&scope, "scope", "n", "", "URN namespace")
```

Comment:
`scope` might not always be URN at the moment.

Reply:
could you suggest an appropriate description?

Reply:
Oh actually I misunderstood what you mean by `URN namespace`, my bad.

We can use `URN's namespace` or keeping it as it is also works for me. wdyt?

Reply:
cool, works